src/components/post-card/PostCard.js

```diff
@@ -59,6 +61,61 @@ export function PostCard(props) {
       }
     </div>
   );
+}*/
+
+export class PostCard extends Component{
+  state = {
+    showComments: false,
+    curValue: 'show comments'
+
+  };
+  toggle = ()=>{
+    this.setState({
+      showComments: !this.state.showComments,
+      curValue: this.state.curValue === 'show comments'? 'hide comments' : 'show comments'
```

Comment:
This approach is good
Also, as another option, you can not to keep label in state and on line 113 check if inside jsx just make a check like this `showComments  ? 'hide' : 'show'`
No need to fix it, just keep in mind for next time